### ISSUE TITLE
Matlab's help function can produce more than one output

### ISSUE THREAD

**Issue by jdbancal** [replab/replab#310]
Running the following command:
```
>> doc ismembc2
Error using help
Too many output arguments.

Error in help2xml (line 59)
[helpstr, docTopic] = help(topic, helpCommandOption);
(...)
```
reveals that some internal functions of matlab can call the `help` function expecting more than one output element. Since our `help` function does not produce any output, matlab cannot call it. We should allow for output arguments and propagate them if needed.

**Comment by jdbancal** [comment]
Actually, matlab can also call `help` with several arguments, so we need to manage also passing more than one argument to the `help` function

**Comment by denisrosset** [comment]
Ouch.

**Comment by denisrosset** [comment]
@jdbancal are you convinced this is fixed now?

**Comment by jdbancal** [comment]
Yes it is fixed!
I thought saying "Fixes #310" in #311 would close this upon merging, but apparently not... let's do it by hand!
